src/main/java/org/eclipse/uprotocol/transport/UMessageReceiver.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.eclipse.uprotocol.transport;
+
+import org.eclipse.uprotocol.status.datamodel.UStatus;
+
+import io.cloudevents.CloudEvent;
+
+
+/** 
+ * uP-L1 Message Receiver.
+ * Used by transports that implement push delivery method to receive 
+ * events asynchronously. 
+ */
+public interface UMessageReceiver {
+
+	/**
+	 * Receive Message Handler
+	 * When messages are received, this function is called.
+	 * @param message uProtocol message represented as bytes 
+	 * @return void
```

Comment:
it returns a UStatus - explain what is returned and what it represents. I think it means your message listener is ready and will be called when messages eventually arrive